src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -229,4 +232,22 @@ private String mangleUrl(String url) {
 
         return url;
     }
+
+    private static String readUrl(String urlString) throws Exception {
+        BufferedReader reader = null;
+        try {
+            URL url = new URL(urlString);
+            reader = new BufferedReader(new InputStreamReader(url.openStream()));
```

Comment:
Possible to use this?
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html